layouts/partials/fragments/nav.html

```diff
@@ -98,17 +98,21 @@
     </div>
   {{- end }}
 </nav>
-{{- if and .Params.breadcrumb (lt (.Params.breadcrumb_level | default 1) (sub (len ($sections | default slice)) (cond (eq $.root.Kind "home") 1 0))) }}
-  <nav aria-label="breadcrumb" class="overlay navbar navbar-expand-lg
-    {{- printf " bg-%s" $bg -}}
+{{- $breadcrumb := .Params.breadcrumb -}}
+{{- $display_breadcrumb := and $breadcrumb $breadcrumb.display -}}
```

Comment:
Why the specific `and` usage here?

Reply:
I rewrote this entire part of code. It was entirely wrong :D